app/react/translations/edit/editors/ArrayEditor/index.js

```diff
@@ -22,7 +22,8 @@ type PropTypes = {
 
 const parseArray = (propValue: any, propKey: string) => {
   switch (propKey) {
-    case 'value': return JSON.parse(propValue);
+    case 'value':
+      return propValue && JSON.parse(propValue);
```

Comment:
It will return `undefined`, `false`, `null` if propValue is not provided. Is it OK?